source/setup.go

```diff
@@ -102,6 +105,10 @@ func (s *Setup) Validate() error {
 		return fmt.Errorf("no source status context provided")
 	}
 
+	if len(s.Scopes) == 0 {
+		return fmt.Errorf("no scopes provided")
```

Comment:
This is pretty minor but could we update this error message to mimic the other ones?

```suggestion
		return fmt.Errorf("no source scopes provided")
```